app/views/external_users/claims/supporting_documents/_new.html.haml

```diff
@@ -12,9 +12,11 @@
   %fieldset
     %legend
       = t('.legend')
```

Comment:
Orphaned fieldset, consider its usage or change to a heading to reflect the section